src/lambdas/GetInstanceUtilization/index.mjs

```diff
@@ -8,24 +8,25 @@ function naiveRound (num, decimalPlaces = 0) {
   return Math.round(num * p) / p
 }
 
-async function getInstanceUtilization (instanceId, region) {
+async function getInstanceUtilization (id, region, Namespace, MetricName, DimensionName) {
   const client = new CloudWatchClient({ region })
   const today = new Date()
   today.setHours(0, 0, 0, 0)
-  const yesterday = new Date(new Date().setDate(new Date().getDate() - 1))
+  const yesterday = new Date()
```

Comment:
Just for readability, I suggest declaring this as a `let` instead of as a `const` since the value changes (even if it is one of its properties).